### PR TITLE
Three small changes

### PR DESCRIPTION
**Pull request by EthanVieira**
* clan members only show examine when clicked
 * up key types previously entered message
 * enter opens and closes chat
     * Currently allowed to open during shop scene. We can discuss this

### CODE REVIEW

File: src/ui/chat-window.js [hunk 1]
```diff
@@ -476,26 +475,42 @@ export class ChatScene extends Phaser.Scene {
...
                 this.userMessage.text =
                     this.userMessage.text.substr(0, this.userMessage.text.length - 2) +
                     "*";
             }
 
+            if (event.keyCode == Phaser.Input.Keyboard.KeyCodes.UP) {
+                this.userMessage.text = this.previousMessage;
+            }
+
+            // There are some firefox only keycodes that we might want to look into
```
Comment (CurtisGreen) [inline src/ui/chat-window.js:496]: Like what?

File: src/ui/chat-window.js [hunk 2]
```diff
@@ -453,9 +453,8 @@ export class ChatScene extends Phaser.Scene {
         }
 
         this.chatInput = this.input.keyboard.on("keydown", (event) => {
-            // enter
-            if (event.keyCode == 13) {
-                if (this.userMessage.text != "") {
+            if (event.keyCode == Phaser.Input.Keyboard.KeyCodes.ENTER) {
```
Comment (CurtisGreen) [inline src/ui/chat-window.js]: Object string is pretty long, maybe pre-declare the keycodes so it's cleaner and can be used for all of the next ones

```js
const KeyCodes = Phaser.Input.Keyboard.KeyCodes;
if (event.KeyCode == KeyCodes.ENTER)
    // ...
```

File: src/ui/chat-window.js (hunk 1 above)
Comment (EthanVieira) [reply src/ui/chat-window.js:496]: At the bottom of this file: https://github.com/photonstorm/phaser/blob/v3.51.0/src/input/keyboard/keys/KeyCodes.js

File: src/ui/chat-window.js (hunk 2 above)
Comment (EthanVieira) [reply src/ui/chat-window.js]: Updated!

**Review by CurtisGreen — APPROVED** [review]

File: src/ui/chat-window.js (hunk 1 above)
Comment (CurtisGreen) [reply src/ui/chat-window.js:496]: Gotcha, can add them later if necessary but right now I don't think it's a big deal

**Pull request merged**